src/theme/DocCard/styles.module.css

```diff
@@ -0,0 +1,57 @@
+.cardContainer {
+  --ifm-link-color: var(--ifm-color-emphasis-800);
+  --ifm-link-hover-color: var(--ifm-color-emphasis-700);
+  --ifm-link-hover-decoration: none;
+
+  box-shadow: 0 1.5px 3px 0 rgb(0 0 0 / 15%);
```

Comment:
wtf 🤔 let's just use `rgba(0, 0, 0, 0.15)` here if it's the same (i sure hope it is?)